coalib/bears/Bear.py

```diff
@@ -388,8 +386,10 @@ def download_cached_file(self, url, filename):
         self.info('Downloading {filename!r} for bear {bearname} from {url}.'
                   .format(filename=filename, bearname=self.name, url=url))
 
-        with urlopen(url) as response, open(filename, 'wb') as out_file:
-            copyfileobj(response, out_file)
+        response = requests.get(url, stream=True, timeout=20)
+        with open(filename, 'wb') as file:
+            for chunk in response.iter_content(125):
```

Comment:
Was there a chunk size that you thought was appropriate?  Comments on https://github.com/coala/coala/issues/3811 would be helpful.